### PR TITLE
Clarify instructions

### PR DESCRIPTION
**Pull request by joshhting**

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -14,26 +14,26 @@ You can, of course, also flag down an instructor if you run into anything.
...
-- [ ] Commit a file called `stuff1.txt` in `b1`.
-- [ ] Commit a file called `stuff2.txt` in `b2`.
-- [ ] Take the changes from `b1` and apply them to `b2`.
-- [ ] In `b2`, delete both files and commit the changes.
-- [ ] Take the changes from `b2` and apply them to `b1`.
-- [ ] Take the changes from `b1` and apply them to the `gburdell3/add-text` branch (this should not cause a merge conflict).
-- [ ] Delete all local branches except for `master` and `gburdell3`.
-- [ ] Send your committed changes to the `gburdell3` branch on your fork
-- [ ] Create a pull request from your fork’s `gburdell3` branch to our master branch and ask one of the instructors in your session to approve it (if you try to merge it you won’t be able to because permissions). **In your pull request description, add a line that says "Fixes #xx" where #xx is the number of the issue you created.** This will auto-close your issue once your pull request is merged.
+1. [ ] Star this repository.
```
Comment (jgkamat) [inline ISSUE_TEMPLATE.md]: I don't think github supports numbers for checkboxes, can you double check that they work? They don't work for me.

Comment (kberzinch) [reply ISSUE_TEMPLATE.md]: The checkboxes still work but the numbers don't show up, so I suggest just leaving as bullets.

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -14,26 +14,26 @@ You can, of course, also flag down an instructor if you run into anything.
...
-- [ ] Commit a file called `stuff1.txt` in `b1`.
-- [ ] Commit a file called `stuff2.txt` in `b2`.
-- [ ] Take the changes from `b1` and apply them to `b2`.
-- [ ] In `b2`, delete both files and commit the changes.
-- [ ] Take the changes from `b2` and apply them to `b1`.
-- [ ] Take the changes from `b1` and apply them to the `gburdell3/add-text` branch (this should not cause a merge conflict).
-- [ ] Delete all local branches except for `master` and `gburdell3`.
-- [ ] Send your committed changes to the `gburdell3` branch on your fork
-- [ ] Create a pull request from your fork’s `gburdell3` branch to our master branch and ask one of the instructors in your session to approve it (if you try to merge it you won’t be able to because permissions). **In your pull request description, add a line that says "Fixes #xx" where #xx is the number of the issue you created.** This will auto-close your issue once your pull request is merged.
+[ ] 1. Star this repository.
```
Comment (jgkamat) [inline ISSUE_TEMPLATE.md]: Checkboxes dont work for me without a bullet in front of them, can you double check this works?
2017-11-06-230837_845x334_scrot

Comment (joshhting) [reply ISSUE_TEMPLATE.md]: Oops

**Pull request merged**